project.clj

```diff
@@ -1,6 +1,6 @@
 (def jetty-version "9.4.11.v20180605")
 
-(defproject puppetlabs/trapperkeeper-webserver-jetty9 "2.5.0-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper-webserver-jetty9 "2.4.0-SNAPSHOT"
```

Comment:
I'm not sure how this got set too high - I can pull this in to a separate PR, or just leave it if it's not a big deal?

Reply:
Hmm, release job may have been set to bump `:minor` instead of `:patch`. We should change that to 2.4.1-SNAPSHOT, which comes before 2.4.1.